src/components/dls/HelperTooltip/HelperTooltip.tsx

```diff
@@ -0,0 +1,21 @@
+import QuestionMarkIcon from '../../../../public/icons/question-mark.svg';
+
+import styles from './HelperTooltip.module.scss';
+
+import Tooltip from 'src/components/dls/Tooltip';
+
+interface HelperTooltipProps {
+  children: React.ReactNode;
+}
+
+const HelperTooltip = ({ children }: HelperTooltipProps) => {
```

Comment:
Looks good overall mA, two things here: 

- can we use an icon with a grey fill similar to canny.io/pricing
- it looks like the icon is not vertically centered (looking at the continue reading example). Can we add the property to the style of the component?